🐜[Bugfix] Resolve loafwallet core update issue

kcw-grunt opened the pull request:
Closes #207

kcw-grunt committed:
🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

kcw-grunt committed:
[Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

kcw-grunt committed:
Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

kcw-grunt committed:
Create sync-mirror.yml

kcw-grunt committed:
Changed the clone sig

kcw-grunt committed:
Update sync-mirror.yml

refactor

kcw-grunt committed:
Update sync-mirror.yml

fixed uname

kcw-grunt committed:
Update sync-mirror.yml

kcw-grunt committed:
Update sync-mirror.yml

kcw-grunt committed:
🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

kcw-grunt committed:
buiid bump (#125)

kcw-grunt committed:
🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

kcw-grunt committed:
🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

kcw-grunt committed:
reemvoed conflict tags

Iferencak committed:
Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

kcw-grunt committed:
changed the urtl

- change the deployment target

Iferencak committed:
Retired old https server code (#154)

Iferencak committed:
#148 Fixed a crash that appeared when the users wallet was aout of sync and a prompt neaded to appear

Iferencak committed:
Removed dispatchqueue.async that was used for testing

kcw-grunt committed:
🚀[Release v3.10.1] Merge into Develop (#162)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* build bump

* clean up duplicate scripts

* readded "Send.UnstoppableDomains.udSystemError" = "System lookup problem. [Error: %2$d]";

-bumped the version and build to test Bitrise

* fixed today extension version and build number

* fixed coding norm (space after colon)

- fixed Turkish - Italian mixup
- version bump

Iferencak committed:
Add SwiftFormat to the codebase (#159)

* Added SwiftFormat to the run scripts

* Removed --elseOnSameLine from rules as it is on by default

* Added --disable enumNamespaces rule

* Using SwiftFormat to auto-format current code

* Removed --allman rule

* 🚀[Release v3.10.1] Merge into Develop (#162)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* build bump

* clean up duplicate scripts

* readded "Send.UnstoppableDomains.udSystemError" = "System lookup problem. [Error: %2$d]";

-bumped the version and build to test Bitrise

* fixed today extension version and build number

* fixed coding norm (space after colon)

- fixed Turkish - Italian mixup
- version bump

* Added SwiftFormat to the run scripts

* Removed --elseOnSameLine from rules as it is on by default

* Added --disable enumNamespaces rule

* Using SwiftFormat to auto-format current code

* Removed --allman rule

* PartnerData auto-format

* Added marking of unused code back to runscripts

* Delete PubtoPrivate.yml

* Delete sync-mirror.yml

Co-authored-by: Kerry Washington <mrkerrywashington@icloud.com>

kcw-grunt committed:
🤬[Techdebt] #167 add custom user agent++ (#168)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* changed the urtl

- change the deployment target

* removed staging server url

normalizing to the prod server only

* aded small test

- build bump

* Retired old https server code (#154)

* Updated

* 🚀[Release v3.10.1] Merge into Develop (#162)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* build bump

* clean up duplicate scripts

* readded "Send.UnstoppableDomains.udSystemError" = "System lookup problem. [Error: %2$d]";

-bumped the version and build to test Bitrise

* fixed today extension version and build number

* fixed coding norm (space after colon)

- fixed Turkish - Italian mixup
- version bump

* added custom var

* build bump

* Add codesigndoc ignore

* Added Swift Packages to the project

- Closes : https://github.com/litecoin-foundation/litewallet-ios/issues/156

* Removed pods

* Added Litewallet PartnerAPI to project

* SUCCESS!

- Added PartnerAPI as a Submodule

* Rebase from SwiftFormat

* Update .gitignore

* Removed submodule

* resetting add the submodule

* updated config

* Removed PartnerAPI #170

Removed references to the LitewalletPartnerAPI lib
- Compiled the app
- Added LitecoinCard View for CardView
- Added CardWebViewController
- Deleted unused views and viewModels

* change build number

* Rmeoved all LitecoinCard Tests

* build bump

* Failed Test for Bitrise CI

* Unbroken tests

* small change build bump

* another build bump

* build again

* build bump

* yet again bump

* small change

* matching build number with bitrise run

* nonsense change

* Fixed the UDSystem Error issues

- removed commented out PartnerAPI Code

Co-authored-by: Iferencak <iferencak@profico.hr>

kcw-grunt committed:
Delete sync-mirror.yml (#176)

kcw-grunt committed:
Delete push-to-release-email.yml (#175)

kcw-grunt committed:
Delete github-actions.yml (#174)

deleting unused workflow

kcw-grunt committed:
Update README.md (#173)

Added a reminder to cleanup Old workflow runs

Iferencak committed:
improved language switching, fixed transparent navigation bar bug (#179)

* Improved language selection

* Fixed transaction completion localization, removed force unwrapings, fixing typos

* autoformat

- BartyCrouch has a bug where the NSLocalizaedString makes extractLocStrings fail.
The workaround is detailed here:
https://github.com/FlineDev/BartyCrouch#exclude-specific-views--nslocalizedstrings-from-localization

* resolved flaky compile issue

- BC and extractLocStrings did not get along
Barty Crouch have an open issue as extractLocStrings is deprecated by Apple so the script is barfing.
The notes on the issue are here: https://github.com/FlineDev/BartyCrouch/issues/135#issuecomment-772379904

The other issue was the NSNotificationCenter was not happy and the work around was:
- deinit and remove observers for the new post
- remove the assertFailure (which I added) and replaced it with an LWAnalytics error event
- added a Error NSlog message
- also moved all the NSN to its own file NSNExtension

* bump version

- polished language view

Co-authored-by: kcw-grunt <mrkerrywashington@icloud.com>

kcw-grunt committed:
🚀[Release v3.11.0] Merge into Develop (#199)

* version bump

- build bump

* build bump to trigger bitrise

* bump for matching bitrise

* Fixed a crash when force trying to get a creation date of unexisting file (#188)

Co-authored-by: Iferencak <iferencak@profico.hr>

kcw-grunt committed:
bump to match current prod build number

Iferencak committed:
Removed app install date variable (#200)

kcw-grunt committed:
nit…format (#203)

kcw-grunt committed:
move the git submodule to the latest

- v1.0.0 releease of loafwallet-core

kcw-grunt committed:
WIP: Not able to type and display recv addresses

- There is a failiure in the  conversion from CChar to Int8
- Have send working

Iferencak commented on loafwallet/src/WalletManager.swift:
```diff
@@ -64,10 +64,11 @@ class WalletManager: BRWalletListener, BRPeerManagerListener {
 		guard let wallet = self.wallet else { return nil }
 		return BRPeerManager(wallet: wallet, earliestKeyTime: self.earliestKeyTime,
-		                     blocks: self.loadBlocks(), peers: self.loadPeers(), listener: self)
+		                     blocks: self.loadBlocks(), peers: self.loadPeers(), listener: self, falsePositivesRate: 0.00001)
```
just wondering, what does the false positive rate do?

kcw-grunt replied:
@Iferencak sorry for the terrible response time. fpRate tells the node to send a percentage of erroneous addresses to Litewallet during sync so that the sent `cloud` is not related to closely to the users actual address. 

The upside is more anonymity (higher rate) the downside is longer time to sync.

josikie reviewed (approved):
Great!
What this code means: return String(NSString(characters: tuplesArray, length: length)) ?
And why do we have # before if and endif?

kcw-grunt committed:
updated loafwallet-core v1.1.0

kcw-grunt committed:
WIP: Added falsePositived interface

- Added enum and set values for
 - low privacy
 - semi-private
 - anonymous
- Updated the loafwallet-core

kcw-grunt committed:
WIP adds the fpRate var

kcw-grunt committed:
Resolved PeerManager set with proper optional setting.

kcw-grunt committed:
WIP: Cannot unwrap ltc1 addresses to retain them in the UILabel

- ltc1 addresses are present but not unwrapping properly
@Iferencak this is the disappearing error you saw when we updated the loafwallet-core.
- Not able to figure this out yet.
- All ideas are welcome

kcw-grunt committed:
Revert "WIP: Cannot unwrap ltc1 addresses to retain them in the UILabel"

This reverts commit 08811b130b0907d77ece6c416359a98be6e08898.
